content/en/docs/concepts/scheduling-eviction/assign-pod-node.md

```diff
@@ -492,6 +498,24 @@ overall utilization.
 Read [Pod topology spread constraints](/docs/concepts/scheduling-eviction/topology-spread-constraints/)
 to learn more about how these work.
 
+## Operators
+
+The following are all the logical operators that you can use with the `operator` field for `nodeAffinity` and `podAffinity` mentioned above.
+
+|    Operator    |    Behaviour    |
+| :------------: | :-------------: |
+| In | The label value is contained in the supplied set of strings |
+|   NotIn   | The label value is not contained in the supplied set of strings |
+| Exists | A label with this key exists on the object |
+| DoesNotExist | No such label with this key exists on the object |
+| Gt | The supplied value will be parsed as an integer, and that integer is less than or equal to the integer that results from parsing the value of a label named by this selector | 
+| Lt | The supplied value will be parsed as an integer, and that integer is greater than or equal to the integer that results from parsing the value of a label named by this selector | 
```

Comment:
These couple of operators can't be used for pod affinity. Maybe put them in a separate table with a preface indicating that they only apply to node affinity.